Formula/omniorb.rb

```diff
@@ -33,7 +33,7 @@ def install
     system "make", "install"
 
     resource("bindings").stage do
-      system "./configure", "--prefix=#{prefix}", "PYTHON=python3"
+      system "./configure", "--prefix=#{prefix}", "PYTHON=#{Formula["python@3.10"].opt_bin/"python3"}"
```

Comment:
This isn't necessary.
```suggestion
      system "./configure", "--prefix=#{prefix}", "PYTHON=python3"
```